setup.py

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env python3
 
 from setuptools import setup
+import sys
+
+if sys.version_info < (3, 3):
+    raise RuntimeError('totp-cli requires python >= 3.3, but this python is %d.%d' % sys.version_info[0:2])
```

Comment:
Oh, that's even better than my suggestion, thanks :+1:

Please wrap this to no more than 80-characters per line (I haven't configured these sort of tests on github, apparently). Other than that looks great.

Thanks,